src/meshes/t8code_mesh.jl

```diff
@@ -427,6 +427,10 @@ function T8codeMesh(trees_per_dimension; polydeg = 1,
                     periodicity = true)
     @assert ((coordinates_min === nothing)===(coordinates_max === nothing)) "Either both or none of coordinates_min and coordinates_max must be specified"
 
+    if coordinates_min !== nothing && coordinates_max !== nothing
+        coordinate_min_max_check(coordinates_min, coordinates_max)
+    end
```

Comment:
```suggestion
    coordinate_min_max_check(coordinates_min, coordinates_max)
```
If you accept my proposal from above.